#Parameters for recognition:
bin_size: 0.1
maxProjectionAngleDeviation: 30
